lib/sikuli/common.sikuli/common.py

```diff
@@ -125,12 +125,26 @@ def _wait_for_loaded(self, component, similarity=0.70):
         @param component: The waiting component
         @param similarity: The similarity of component. Default: 0.70.
         """
+        is_exists = False
         for counter in range(50):
```

Comment:
5 second might be okay for gmail. However, in FB, I encountered a case that it was delayed for around 10 seconds. Can you try to  make the function `_wait_for_loaded(self, component, time=5, similarity=0.7)` & `il_type(self, message, width, height, wait_component=None, time=5)` and `range(time*10)` in the for loop below please? In that case, I can reuse it in FB cases and other cases.

Thank you.